pkgs/development/python-modules/tokenizers/default.nix

```diff
@@ -60,19 +61,22 @@ in rustPlatform.buildRustPackage rec {
     hash = "sha256-rQ2hRV52naEf6PvRsWVCTN7B1oXAQGmnpJw4iIdhamw=";
   };
 
-  cargoSha256 = "sha256-BoHIN/519Top1NUBjpB/oEMqi86Omt3zTQcXFWqrek0=";
+  cargoDeps = fetchCargoTarball {
+    inherit src sourceRoot;
+    sha256 = "sha256-BUiF89lw/QCbifGVD50VpKUl/U+0Kz/508nSaCAdi2k=";
+  };
 
   sourceRoot = "source/bindings/python";
 
   nativeBuildInputs = [
-    pipInstallHook
+    cargo
```

Comment:
cargo and rustc are always required for the hook, right? If so we can include them by default like pytestCheckHook included pytest to make the hook easier to use and remove the inherit.

Reply:
Not for the setup hook. They are required by the actual build, so they should probably be propagated native inputs from `setuptools-rust`.

Reply:
I thought of that first too, but then you need to select a rust/cargo version, and I think we should avoid that.